sources/engine/Stride.Physics/Data/ConvexHullColliderShapeDesc.cs

```diff
@@ -49,6 +49,12 @@ public class ConvexHullColliderShapeDesc : IAssetColliderShapeDesc
         [DataMember(45)]
         public Vector3 Scaling = Vector3.One;
 
+        /// <userdoc>
+        /// The Margin of the generated convex hull.
+        /// </userdoc>
+        [DataMember(48)]
+        public float Margin { get; set; }
```

Comment:
Just a small note, it looks like margins in bullet by default should be around `0.04` for dynamic and `0.0` for static objects. Convex hulls are likely used as replacement for the lack of support of non-static meshes, so best set it to the default value for dynamics, so `0.04` by default.

Reply:
Good call, adding that now